rootchain/sampool/bucket.go

```diff
@@ -0,0 +1,63 @@
+package sampool
+
+import (
+	"github.com/0xPolygon/polygon-edge/rootchain"
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+type samSet struct {
+	messages   []rootchain.SAM
+	signatures map[string]bool
+}
+
+func newUniqueSAMs() samSet {
```

Comment:
What does `unique` mean? Do you need it in the function name?

Reply:
Leftover code. 

[Renamed](https://github.com/0xPolygon/polygon-edge/pull/718/commits/fe6278544c8f62a6f5938e8b3f7fb66f53ea94c8)